addons/hr_holidays/i18n/hr_holidays.pot

```diff
@@ -3781,6 +3781,13 @@ msgstr ""
 msgid "Time Off to Approve"
 msgstr ""
 
+#. module: hr_holidays
+#. odoo-python
+#: code:addons/hr_holidays/models/hr_leave.py:0
+#, python-format
+msgid "Time Off type changed from %s to %s"
+msgstr ""
+
```

Comment:
Is it required?

Reply:
Nope, i forgot to remove it when committing.